Makefile

```diff
@@ -26,23 +26,34 @@ refman-release: exe
 
 z3-mac:
 	mkdir -p ${DIR}Binaries/z3/bin
-	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-02-17/z3-4.12.1-macos-11-bin.zip
-	unzip z3-4.12.1-macos-11-bin.zip
-	rm z3-4.12.1-macos-11-bin.zip
-	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-02-17/z3-4.8.5-macos-11-bin.zip
-	unzip z3-4.8.5-macos-11-bin.zip
-	rm z3-4.8.5-macos-11-bin.zip
+	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-08-02/z3-4.12.1-x64-macos-11-bin.zip
+	unzip z3-4.12.1-x64-macos-11-bin.zip
+	rm z3-4.12.1-x64-macos-11-bin.zip
+	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-08-02/z3-4.8.5-x64-macos-11-bin.zip
+	unzip z3-4.8.5-x64-macos-11-bin.zip
+	rm z3-4.8.5-x64-macos-11-bin.zip
+	mv z3-* ${DIR}/Binaries/z3/bin/
+	chmod +x ${DIR}/Binaries/z3/bin/z3-*
+
+z3-mac-arm:
+	mkdir -p ${DIR}Binaries/z3/bin
+	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-08-02/z3-4.12.1-arm64-macos-11-bin.zip
+	unzip z3-4.12.1-arm64-macos-11-bin.zip
+	rm z3-4.12.1-arm64-macos-11-bin.zip
+	wget https://github.com/dafny-lang/solver-builds/releases/download/snapshot-2023-08-02/z3-4.8.5-arm64-macos-11-bin.zip
```

Comment:
@atomb These 3 lines are not correct, you need to replace z3-4.8.5-arm64 with z3-4.8.5-x64! Not sure how to do it by myself when PR is already merged.

Reply:
Oh, good catch! I'll fix it in a separate PR.